components/Layout/Layout.tsx

```diff
@@ -0,0 +1,43 @@
+import Head from 'next/head';
+import styles from './Layout.module.scss';
+import type LayoutProps from './types';
+import HeadingSlackCTA from './HeadingSlackCTA';
+import FooterSlackCTA from './FooterSlackCTA';
+import Footer from './Footer';
+
+const Layout = ({ title, children, lastPublishedAt }: LayoutProps) => (
+    <>
+        <Head>
+            <title>{title}</title>
```

Comment:
Nice positioning of `Head`, maybe add a ticket to codebase as a reminder to add social media metadata here later on.

Reply:
I've added a ticket for adding social media metadata to Codebase.